showcase/src/app/demo-ui/components/table-with-items-per-page-selector/table-with-items-per-page-selector.component.ts

```diff
@@ -0,0 +1,97 @@
+import { Component, Inject } from "@angular/core";
```

Comment:
Same here, could you please add an example of this file in the "showcase/src/assets/examples/table/..." ?